www/js/session.js

```diff
@@ -27,7 +27,24 @@ define([
 		}
 		return Number(hash).toString(16);
 	};
-
+	var sayswho = (function(){
+	    var ua= navigator.userAgent, tem,
+	    M= ua.match(/(opera|chrome|safari|firefox|msie|trident(?=\/))\/?\s*(\d+)/i) || [];
+	    if(/trident/i.test(M[1])){
+	        tem=  /\brv[ :]+(\d+)/g.exec(ua) || [];
+	        return 'IE '+(tem[1] || '');
+	    }
+	    if(M[1]=== 'Chrome'){
+	        tem= ua.match(/\bOPR\/(\d+)/)
+	        if(tem!= null) return 'Opera '+tem[1];
+	    }
+	    M= M[2]? [M[1], M[2]]: [navigator.appName, navigator.appVersion, '-?'];
+	    if((tem= ua.match(/version\/(\d+)/i))!= null) M.splice(1, 1, tem[1]);
+	    return {
+			name: M[0],
+			version: M[1]
+		};
```

Comment:
u have this code here then u are checking only the browsers in main.js. Then my suggestion would be to put the check here as well and set a variable which will tell you all this and u can keep using that, and can move the check out and just show the message in main.js

Reply:
Moved the check from main.js to session.js